src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -110,6 +110,19 @@ public void add(Person toAdd) throws DuplicatePersonException {
         internalList.add(toAdd);
     }
 
+    /**
+     * Edits a person in the list
+     */
+
+    public void edit(Person toEdit, ReadOnlyPerson targetPerson) throws DuplicatePersonException {
```

Comment:
Is this method really editing a person in the list? Maybe make the header more descriptive.